packages/cli/src/lib/errors/CLIBaseError.ts

```diff
@@ -0,0 +1,6 @@
+export class CLIBaseError extends Error {
+  public name = 'CLI Error';
+  public constructor(message?: string) {
+    super(message);
+  }
+}
```

Comment:
I think you should make this an abstract class ;)
```suggestion
export abstract class CLIBaseError extends Error {
  public name = 'CLI Error';
  public constructor(message?: string) {
    super(message);
  }
}
```